### PR TITLE
feat(metrics): Add metrics.users.disabled flag

### PR DESCRIPTION
**Pull request by tfadeyi**
### 📋 Changes

* Introduces the `--metrics.users.disabled` flag, to disable user metrics like the monthly_active_users.
* Updates the helm chart to add the metrics.users.disabled.
* Updates the helm chart docs.

### CODE REVIEW

File: .github/workflows/chart-testing.yaml [hunk 1]
```diff
@@ -62,4 +62,8 @@ jobs:
       with:
         version: 'latest'
     - name: Run chart-testing (install)
-      run: ct install --config chart-testing.yaml
+      run: ct install --config chart-testing.yaml --helm-extra-set-args '--set=exporter.tls.disabled=true --set=auth0.domain="${{ secrets.TEST_AUTH0_DOMAIN }}" --set=auth0.clientId="$CLIENT_ID" --set=auth0.clientSecret="$CLIENT_SECRET"'
```
Comment (hobbsh) [inline .github/workflows/chart-testing.yaml:65]: You can also create `ci/test-values.yaml` in the root of the chart directory and `chart-testing` will use those when it installs. The format is just regular helm values, but only overrides from default values.yaml.

Comment (tfadeyi) [reply .github/workflows/chart-testing.yaml:65]: If I had a `ci/test-values.yaml` is there a way I can populate the values in the yaml with the secrets in CI?
Maybe I could use `envsubst`, but I'm not 100% sure

File: deploy/charts/auth0-exporter/values.yaml [hunk 2]
```diff
@@ -42,6 +42,13 @@ exporter:
   port: 9301
   logLevel: info
 
+  # -- Exporter's metrics configuration
+  metrics:
+    # -- Tenant Users metrics exported by the exporter
+    users:
+      # -- Stops the exporter from fetching user info from the Auth0 tenant
+      disabled: false
```
[the remark this answers is not in the document]
Comment (tfadeyi) [reply deploy/charts/auth0-exporter/values.yaml:50]: Yeah, that's a good point, I'll change it to `enabled: true` and let the user decide to disable it or not

File: .github/workflows/chart-testing.yaml (hunk 1 above)
Comment (hobbsh) [reply .github/workflows/chart-testing.yaml]: Oh I missed that bit - no. The secrets would need to get written as a k8s secret and then referenced in the chart.

File: deploy/charts/auth0-exporter/values.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (tfadeyi) [reply deploy/charts/auth0-exporter/values.yaml:50]: I really misread the comment :laughing:  I'll create a new PR for this

**Pull request merged**